lib/events.js

```diff
@@ -436,6 +436,14 @@ function listenerCount(type) {
   return 0;
 }
 
+EventEmitter.prototype.eventNames = function() {
+  const events = this._events;
+  if (events) {
+    return Object.getOwnPropertyNames(events).concat(
```

Comment:
Isn't `Object.keys()` sufficient here since we aren't creating non-enumerable properties? It should be much faster than `Object.getOwnPropertyNames()` too.